api/src/scripts/updateDoi.ts

```diff
@@ -198,19 +198,12 @@ const updateDoi = async (): Promise<void> => {
         }
     });
 
-    // Simplify publications
-    const simplifiedPublications = publications.map((publication) => {
-        // Discard versionOf field from current version
-        const { versionOf, ...versionRest } = publication.versions[0];
-
-        return { ...versionRest, versionId: versionRest.id, ...publication };
-    });
-
     let index = 1;
 
-    for (const publication of simplifiedPublications) {
-        await helpers.updateDOI(publication.doi, publication, publication.References).catch((err) => console.log(err));
-        console.log(`No: ${index}. ${publication.title} doi updated (${publication.doi})`);
+    for (const publication of publications) {
+        const references = publication.versions[0].References;
```

Comment:
publications.versions[0] may not be the latest published version

Reply:
This should be OK now that we're using isLatestLiveVersion in this file